panther_gpiod/src/gpio_driver.cpp

```diff
@@ -217,7 +236,7 @@ void GPIODriver::MonitorAsyncEvents()
   pollfd.events = POLLIN;
 
   {
-    std::lock_guard<std::mutex> lck(monitor_init_mtx_);
+    std::unique_lock<std::mutex> lck(monitor_init_mtx_);
```

Comment:
I think that lock_guard is better in this case.

Reply:
I don't know the reason this was changed for prealpha. @pidpawel ?

Reply:
something went no yes with wild merging. In this simple case lock_guard should be a better fit.

Reply:
I think all locks in file can be made `lock_guard` correct me if I'm wrong